keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -437,46 +437,42 @@ Operation metrics from [csiOperationsLatencyMetric](https://github.com/kubernete
 
 #### VolumeAttributesClass Deletion Protection
 
-While a VolumeAttributesClass is referenced by any PVC, we will prevent the object from being deleted by adding a finalizer([reference](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/storage/storageobjectinuseprotection/admission.go)). 
+While a VolumeAttributesClass is referenced by any PVC, we will prevent the object from being deleted by adding a finalizer([reference](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/storage/storageobjectinuseprotection/admission.go)). It's a best effort to prevent users from making mistakes. It may not be accurate in all cases.
 
-The **vac_finalizer_controller** sets/removes the finalizer on all VolumeAttributesClasses. This prevents users from deleting a VolumeAttributesClass that's used by a PVC. The vac_finalizer_controller will maintain a cache of a map of VolumeAttributesClass to the list of PVCs that are using the VolumeAttributesClass.
+The existing admission controller plugin named **StorageObjectInUseProtection** will set a finalizer on the object when a request is made to create a VolumeAttributesClass object. The finalizer is removed by the **vac_finalizer_controller** when it's not referenced. 
 
-There are a few conditions that will trigger add/remove pvc finalizers in the VolumeAttributesClass:
+The **StorageObjectInUseProtection** admission plugin is enabled by default, but it's likely disable by users, so the **vac_finalizer_controller** is also responsible for setting the finalizer on the VolumeAttributesClass object if the finalizer is not set when the controller observes the VolumeAttributesClass object or PVC/PV object that references the VolumeAttributesClass object.
```

Comment:
Do we need the admission controller if we're also going to have a controller and this is best effort?